pkg/sql/schemachanger/scbuild/builder.go

```diff
@@ -139,6 +152,47 @@ func (b *buildContext) build(
 	return b.outputNodes, nil
 }
 
+func (b *buildContext) addNodeOnce(dir scpb.Target_Direction, elem scpb.Element) {
```

Comment:
Perhaps I'm wrong but once we approach being "done" with the new schema changer won't it be the case that `addNodeOnce` is going to be used a lot more than `addNode`? It seems (naively perhaps) that it should almost be the default. Perhaps I'm wrong.